src/plugins/bookmark/README.md

```diff
@@ -0,0 +1,27 @@
+# Bookmark
+
+Nonlinear navigation similar to the Goto plugin.
+
+Goto supports nonlinear navigation by *locally* defining *out-links*, accessible via the arrow keys.
+
+Bookmark supports nonlinear navigation by *globally* defining *in-links*, accessible via normal keys like 1,2,3,A,B,C.
+
+Example:
+
+```html
+<!-- data-bookmark-key-list allows an "inbound"-oriented style of non-linear navigation. -->
+<div id="..." class="step" data-bookmark-key-list="Digit1 KeyA 1 2 3 a b c">
+```
+
+An `id` is required on the `div`.
+
+If you assign the same key to multiple steps, that hotkey will cycle among them.
+
+WARNING: It's up to you to avoid reserved hotkeys H, B, P, ?, etc.
```

Comment:
Couldn't you just disallow the usage of those keys?

Reply:
Don't think so, it would clobber the other plugins … I'll think about how to resolve the interaction effects.

Reply:
I think this is ok. If you want to override a reserved key, just do it.